### DISCUSSION TITLE
[no subject alternative dns name matching netbeans.apache.org found. ]Can't download plugin!

### DISCUSSION THREAD

**Discussion by magbroxwt** [apache/netbeans#3929] [Q&A]
When I tried to download plugin , it would fail and shew "Networking problem in https://netbeans.apache.org/nb/updates/12.6/ide/org-netbeans-modules-dlight-nativeexecution-nb.nbm " 
In proxy Configuration , test connection will shows "no subject alternative dns name matching netbeans.apache.org found. "

I've try to find answer, maybe its just because of my local network so poor .
Really wish anyone could help me solve this problem.Thank you!

**Comment by asbachb** [comment]
Are you in a cooperate environment? It sounds like there's some TLS interceptor.

**Reply by magbroxwt** [reply]
Thank you for your time.
I installed c++ plugin successfully once ,but now it failed.
How to find any TLS interceptor? I'm not very clear about it.
I just used SSR to cross some limits and I close it now .

**Reply by magbroxwt** [reply]
I have tried to reboot my computer but it dosen't work .

**Reply by asbachb** [reply]
Are you in a company or is it your private computer at home?

**Reply by magbroxwt** [reply]
yes. Private computer at home
